agent/internal.go

```diff
@@ -51,6 +51,7 @@ type listArchivesResponse struct {
 type startChatRequest struct {
 	Chat       *InitialChat `json:"chat,omitempty"`
 	Continuous bool         `json:"continuous,omitempty"`
+	Active     bool         `json:"active,omitempty"`
```

Comment:
Oh, Mr. Treter! The default value of the param in API is `true` so we have to explicitly call with false and your `omitempty` wouldn't allow for this. Am I right, Mr. Treterinho?

Reply:
Hmmm In fact here in such a situation we should use `*bool` instead of classic as it could be hard to distinguish default value from the user or explicit `false`

Reply:
bullshit but we already talked about it 😂

Reply:
XD